working_time/working_time/doctype/working_time_log/working_time_log.py

```diff
@@ -3,6 +3,26 @@
 
 # import frappe
 from frappe.model.document import Document
+from frappe.utils import time_diff_in_seconds
+
 
 class WorkingTimeLog(Document):
-	pass
+	def cleanup(self):
```

Comment:
```suggestion
	def cleanup_and_set_duration(self):
```
`cleanup` is a little ambiguos in `working_time.py`